CONTRIBUTING.md

```diff
@@ -175,3 +222,146 @@ Here are some examples of common test patterns:
     }
     ```
     Here we are using the `assert_empty` helper command that works with both Helm 2 and 3.
+    
+### Writing Acceptance Tests
+
+If you are adding a feature that fits thematically with one of the existing test suites,
+then you need to add your test cases to the existing test files.
+Otherwise, you will need to create a new test suite.
+
+#### Adding Test Suites
+
+To add a test suite, create a new package directory and add a `main_test.go` file.
+At a minimum, this file needs to contain the following:
+
+```go
```

Comment:
What do you think about actually creating a directory in tests/acceptance called `template` that has these files we can copy/paste?

My reasoning is that folks are more likely to copy/paste from there than from this doc.

Reply:
I like this idea as well.